doctr/utils/lang_detect.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2022, Mindee.
```

Comment:
@aminemindee I think we should move this to https://github.com/mindee/doctr/blob/main/doctr/models/_utils.py and move the corresponding test also

Reply:
Done and pushed.